Sources/StreamChat/WebSocketClient/Events/ChannelEvents.swift

```diff
@@ -4,7 +4,7 @@
 
 import Foundation
 
-public struct ChannelUpdatedEvent: EventWithChannelId, EventWithPayload {
+public struct ChannelUpdatedEvent: ChannelEvent {
     public let cid: ChannelId
     public let userId: UserId?
```

Comment:
I'd remove this since it doesn't have any added value. Also please remove all mentions of invites since we don't support it yet and it's just confusing :)